src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -350,16 +349,17 @@ private void BuildFallbackBlock(TagHelperContent builder)
                 // Build the <script /> tag that checks the effective style of <meta /> tag above and renders the extra
                 // <link /> tag to load the fallback stylesheet if the test CSS property value is found to be false,
                 // indicating that the primary stylesheet failed to load.
+                // GetEmbeddedJavaScript returns JavaScript to which we add '"{0}","{1}",{2});'
                 builder
                     .AppendHtml("<script>")
-                    .AppendHtml(
-                        string.Format(
-                            CultureInfo.InvariantCulture,
-                            JavaScriptResources.GetEmbeddedJavaScript(FallbackJavaScriptResourceName),
-                            JavaScriptEncoder.Encode(FallbackTestProperty),
-                            JavaScriptEncoder.Encode(FallbackTestValue),
-                            JavaScriptStringArrayEncoder.Encode(JavaScriptEncoder, fallbackHrefs)))
-                    .AppendHtml("</script>");
+                    .AppendHtml(JavaScriptResources.GetEmbeddedJavaScript(FallbackJavaScriptResourceName))
+                    .AppendHtml("\"")
+                    .AppendHtml(JavaScriptEncoder.Encode(FallbackTestProperty))
```

Comment:
If these intermediate encoded `string`s turn out to be a significant cost, could create an `IHtmlContent` implementation that's similar to `StringHtmlContent` but ignores the `encoder` passed to `WriteTo()` and instead encodes using a `JavaScriptEncoder` passed to the constructor. Likely a useful follow-up but not something I'm doing today.